pkg/controller/storage/bucket/aws_handler.go

```diff
@@ -18,6 +18,7 @@ package bucket
 
 import (
 	"fmt"
+	"reflect"
 
 	"github.com/aws/aws-sdk-go-v2/service/s3"
 	s3Bucketv1alpha1 "github.com/crossplaneio/crossplane/pkg/apis/aws/storage/v1alpha1"
```

Comment:
Not introduced in this PR, but it's very unusual to have capitolised import names in Go.